Orange/widgets/data/tests/test_owunique.py

```diff
@@ -71,22 +71,22 @@ def test_compute(self):
         w.selected_vars = w.var_model[:2]
 
         w.tiebreaker = "Last instance"
-        w.commit()
+        w.commit.now()
```

Comment:
In tests - in particular in places like this - `commit.now` is more appropriate than `commit.deferred`, which relies on auto apply being on.